### PR TITLE
[pulp] Including settings.local.py file to be collected and parsed.

### PR DESCRIPTION
**Pull request by rbertol**
This change, allows the sosreport to collect the
settings.local.py file, when customer has custom changes out of the default settings.py file.

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [X] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [ ] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?
- [X] Are all passwords or private data gathered by this PR [obfuscated](https://github.com/sosreport/sos/wiki/How-to-Write-a-Plugin#how-to-prevent-collecting-passwords)?

### CODE REVIEW

**Review by arif-ali — APPROVED** [review]

File: sos/report/plugins/pulp.py [hunk 1]
```diff
@@ -72,6 +72,7 @@ def setup(self):
         self.add_copy_spec([
             "/etc/pulp/*.conf",
             "/etc/pulp/settings.py",
+            "/etc/pulp/settings.local.py",
```
Comment (TurboTurtle) [inline sos/report/plugins/pulp.py:75]: Do we need to add this to the `postproc()` handling of the "regular" `settings.py` file?

https://github.com/sosreport/sos/blob/f580ee0719b298d3358fc628a7baa690373a5b31/sos/report/plugins/pulp.py#L182

Comment (rbertol) [reply sos/report/plugins/pulp.py:75]: By default, customer should not include passwords. But will not warm redact it, in case of any sensitive info.

**Comment by rbertol** [comment]
Also, removed the settings.loca.py from the file section, as this file will not be always present on every installation.

**Review by arif-ali — CHANGES_REQUESTED** [review]
Can you fix the flake8 issue please

**Review by arif-ali — APPROVED** [review]

**Review by TurboTurtle — APPROVED** [review]

**Pull request merged**